src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -565,6 +577,12 @@ private async ValueTask<bool> HandleSendChat(ClientPlayer sender, string message
             return !@event.IsCancelled;
         }
 
+        private ValueTask<bool> HandleSendQuickChat(ClientPlayer sender, IQuickChatMessage message)
+        {
+            // TODO add event
+            return new ValueTask<bool>(true);
+        }
```

Comment:
Can we make quick chat cancellable event so that we could use this for some custom things in plugins?

Reply:
Good idea, noted.